python/tank/authentication/defaults_manager.py

```diff
@@ -96,7 +96,10 @@ def get_login(self):
         :returns: Default implementation returns the login for the
                   currently stored user.
         """
-        return session_cache.get_current_user(self.get_host())
+        if self.get_host():
+            return session_cache.get_current_user(self.get_host())
+        else:
```

Comment:
can we add some comments explaining when this state is likely to be hit (e.g. no cache file present)?